test/test_torch.py

```diff
@@ -8334,6 +8334,100 @@ def test_contiguous(self, device):
         x.set_(x.storage(), 0, x.size(), stride)
         self.assertTrue(x.is_contiguous())
 
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
+    @onlyOnCPUAndCUDA
+    @dtypes(torch.float, torch.double, torch.half, torch.int, torch.long, torch.cfloat, torch.cdouble, torch.bool, torch.uint8)
```

Comment:
Please sort these dtypes (uint8->complex128)

Reply:
We also have dtype acquisition helpers here:

https://github.com/pytorch/pytorch/blob/8a1e1004664e4cc8b1adbcef07b9fdae397e1787/torch/testing/__init__.py#L316

Reply:
Changed to `get_all_dtypes`. Just FYI, I'm skipping BFloat because numpy doesn't support it.

Reply:
That's OK. Would you just add a brief comment as to why bfloat16 is skipped at the top of the test?

Reply:
Done